types/ember__application/test/deprecations.ts

```diff
@@ -2,7 +2,8 @@ import { deprecate, deprecateFunc } from '@ember/application/deprecations';
 
 deprecate('this is no longer advised', false, {
     id: 'no-longer-advised',
-    until: 'v4.0'
+    until: 'v4.0',
+    url: 'https://emberjs.com'
```

Comment:
I'm not sure if it's better, but if you split this call into two where one contains the url and the other doesn't, it keeps the old test while testing the new (optional) behavior.

Reply:
Sure, updated it!